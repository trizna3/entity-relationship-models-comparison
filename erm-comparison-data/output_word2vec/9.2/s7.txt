units-units
fuel type-EMPTY
weather-weather
tile-tiles
games-game
boards-board
tile type-EMPTY
