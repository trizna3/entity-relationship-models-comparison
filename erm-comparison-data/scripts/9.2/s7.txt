entity set game {
};

entity set board {
	"rows"
	"columns"
};

entity set tiles {
	"type"
	"coordinate x"
	"coordinate y"
};

entity set weather {
	"type"
};

entity set "points for movement" {
};

entity set units {
	"name"
	"health"
	"attack strength"
	"defense strength"
	"points count"
	"position"
};

entity set fuel {
	"type"
};

association "runs on" between 
{
	board ONE
	game ONE
}{};

association "contains" between 
{
	board ONE
	tiles MANY
}{};

association between 
{
	board ONE
	units MANY
}{};

association between 
{
	weather ONE
	"points for movement" MANY
}{};

association between 
{
	units ONE
	"points for movement" MANY
}{};

association between 
{
	tiles ONE
	"points for movement" MANY
}{};

generalization from fuel to units;