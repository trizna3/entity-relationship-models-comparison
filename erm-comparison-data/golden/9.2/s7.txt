games-game
boards-board
tile-tiles
tile type-
weather-weather
fuel type-fuel
units-units
move-points for movement
REBIND_NARY_ASSOCIATION:E-weather;fuel type;tile type
